Create testing-mobile-update.md (#62)
atolye15/checklist: checklists/testing-mobile-update.md

---
title: Testing Mobile Update
category: Project Management
date: "2020-03-11"
tags: ['test', 'mobile', 'testflight', 'production']
description: Before releasing mobile application, you should test it in different cases. Here are the methods of it!
---

- [ ] Test the application on stage environment  
We do our tests by using TestFlight on iOS and relevant APK on Android. Download the stage application and start testing now! 

> - [ ] Test all the issues that are on Jira for current sprint  

Reviewer: ```suggestion
- [ ] Test all the issues that are on Jira for the ongoing sprint  
```

> - [ ] Test the application in general  

Reviewer: ```suggestion
- [ ] Test the application generally  
```

> In some cases other parts of the apllication might have broken, even anything developed in that parts.

Reviewer: ```suggestion
In some cases other parts of the application might have been broken, even if there was nothing developed in that parts.
```

- [ ] Test older version of the application  
So that you can be sure that everything works on older version of the app correctly. 

> - [ ] Make sure relevant API version has been deployed to production

Reviewer: ```suggestion
- [ ] Make sure relevant API version has been deployed to production  
```

> - [ ] Download the new version of the application from stores

Reviewer: ```suggestion
- [ ] Download the new version of the application from store markets  
```

> - [ ] Apply all the tests on production environment

Reviewer: ```suggestion
- [ ] Apply all the tests on production environment  
```

---
title: Testing Mobile Update
category: Project Management
date: "2020-03-11"

> tags: ['test', 'mobileapp']

Reviewer 2: ```suggestion
tags: ['test', 'mobile', 'production']
```
